app/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
don't want to start a formatting discussion here :) however i wonder if we should have a common set of rules in @bible-engine ? of course all rules here (except `singleQuote` conflict with the style used in the other packages, so i guess this is why you put separate prettifier settings here? if you feel like you want those settings in the app code that's ok. apart from that however agreeing on a commons style would make working/contributing on the repo easier.

Reply:
common settings are good, but I the 2 space tabs are really useful for large components with deep html hierarchy. With 4 tabs, ends up getting pretty far to the right. That was the main issue